pkg/common/jwtsvid/validate_test.go

```diff
@@ -0,0 +1,227 @@
+package jwtsvid
```

Comment:
This is just a renamed token_test.go with just the validation test portion remaining.